cmd/minikube/cmd/docker-env.go

```diff
@@ -287,14 +288,23 @@ docker-cli install instructions: https://minikube.sigs.k8s.io/docs/tutorials/doc
 		if len(co.Config.Nodes) > 1 {
 			exit.Message(reason.EnvMultiConflict, `The docker-env command is incompatible with multi-node clusters. Use the 'registry' add-on: https://minikube.sigs.k8s.io/docs/handbook/registry/`)
 		}
+		cr := co.Config.KubernetesConfig.ContainerRuntime
+		// nerdctld supports amd64 and arm64
+		if err := dockerEnvSupported(cr, driverName); err != nil {
+			exit.Message(reason.Usage, err.Error())
+		}
 
-		if co.Config.KubernetesConfig.ContainerRuntime != constants.Docker {
-			exit.Message(reason.Usage, `The docker-env command is only compatible with the "docker" runtime, but this cluster was configured to use the "{{.runtime}}" runtime.`,
-				out.V{"runtime": co.Config.KubernetesConfig.ContainerRuntime})
+		// for the sake of docker-env command, start nerdctl and nerdctld
+		if cr == constants.Containerd {
+			out.WarningT("Using the docker-env command with the containerd runtime is a highly experimental feature, please provide feedback or contribute to make it better")
```

Comment:
this will Break the Eval $()
it is better to skip this... or maybe put it a comment

# this is an experimental feature for containerd
export DOCKER_TLS_VERIFY="1"
export DOCKER_HOST="tcp://127.0.0.1:56081"
export DOCKER_CERT_PATH="/Users/medya/.minikube/certs"
export MINIKUBE_ACTIVE_DOCKERD="minikube"

Reply:
Actually I think it won't break the eval. out. `WarningT` prints things to stderr. It does not break eval, which only use stdout.

You can try to execute the `eval $(minikube -p minikube docker-env --ssh-host)`, it sets env correctly while show the warning in stderr.
```shell
eval $(minikube -p minikube docker-env --ssh-host)
```

Reply:
Done. I make ssh-host and ssh-add become true automatically when they are using docker-env on containerd.

Besides, I use out.WarningT  to remind user that they must execute 'ssh-agent bash' and 'minikube ssh-host --append-known' before minikube docker-env. out.WarningT will not break the eval